mscore/shortcut.cpp

```diff
@@ -1757,7 +1757,17 @@ Shortcut Shortcut::_sc[] = {
          MsWidget::MAIN_WINDOW,
          STATE_NORMAL | STATE_NOTE_ENTRY | STATE_EDIT | STATE_PLAY,
          "toggle-navigator",
-         QT_TRANSLATE_NOOP("action","Navigator"),
+         QT_TRANSLATE_NOOP("action","Navigator")//,
+         //0,
+         //0,
+         //Icons::Invalid_ICON,
+         //Qt::ApplicationShortcut
```

Comment:
You removed the Default shortcut, but here you remove the ability to assign one, I think?